packages/jest-resolve/src/index.js

```diff
@@ -108,13 +108,12 @@ class Resolver {
     return null;
   }
 
-  resolveModule(
-    from: Path,
+  _resolveModuleFromDirIfExists(
```

Comment:
Can we make this public (remove the `_`) with this signature so we can properly rely on it in `jest-runtime` or does someone have concerns over the signature/naming?